gradio/inputs.py

```diff
@@ -67,7 +67,7 @@ def rebuild_flagged(self, dir, msg):
         """
         All interfaces should define a method that rebuilds the flagged input when it's passed back (i.e. rebuilds image from base64)
         """
-        pass
+        return msg
```

Comment:
I thought `msg` is always blank?

Reply:
See above for `msg`/`message` confusion. changed `msg` to `data`.